pennylane_lightning/core/src/simulators/lightning_qubit/gates/KernelMap_X64.cpp

```diff
@@ -84,4 +84,22 @@ int assignKernelsForMatrixOp() {
     // LCOV_EXCL_STOP
     return 1;
 }
+int assignKernelsForControlledGateOp() {
+    assignKernelsForControlledGateOp_Default();
+
+    // LCOV_EXCL_STOP
```

Comment:
why only the ``LCOV_EXCL_STOP`` part?

Reply:
Good question. I think that's faulty copy-pasting. Will fix.